rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -856,6 +857,50 @@ def test_upgrade_v8_v9(globaldb: GlobalDBHandler, messages_aggregator):
         ).fetchone()[0] == 0
 
 
+@pytest.mark.parametrize('custom_globaldb', ['v9_global.db'])
+@pytest.mark.parametrize('target_globaldb_version', [9])
+@pytest.mark.parametrize('reload_user_assets', [False])
+def test_upgrade_v9_v10(globaldb: GlobalDBHandler, messages_aggregator):
+    """Test upgrade from v9 to v10 which adds is_main_asset field to multiasset_mappings"""
+    with globaldb.conn.read_ctx() as cursor:
+        assert not column_exists(
+            cursor=cursor,
+            table_name='multiasset_mappings',
+            column_name='is_main_asset',
+        )
+        mappings_count = cursor.execute('SELECT COUNT(*) FROM multiasset_mappings').fetchone()[0]
+        # get lexicographically first asset for collection 40 before upgrade
+        collection_40_first = cursor.execute('SELECT asset FROM multiasset_mappings WHERE collection_id=40 ORDER BY asset LIMIT 1').fetchone()[0]  # noqa: E501
```

Comment:
Hmm the check I would want to see in this test is:

1. Before upgrade take all groups
2. After upgrade make sure the same amount of groups is there
3. Adn that also for every group the first asset is main asset
4. Except for the dAI one.

Also the schema changes but that you already did.